packages/legacy/core/App/screens/ProofRequest.tsx

```diff
@@ -262,6 +268,10 @@ const ProofRequest: React.FC<ProofRequestProps> = ({ navigation, route }) => {
         proofRecordId: proof.id,
         proofFormats: automaticRequestedCreds.proofFormats,
       })
+
+      if (proof.connectionId && goalCode && goalCode.endsWith('verify.once')) {
+        agent.connections.deleteById(proof.connectionId)
```

Comment:
Does this cause the contact to be removed or do they still appear in a wallet holder's contact list after this?

Reply:
Yes this removes it from the contact list screen